README.md

```diff
@@ -140,7 +140,8 @@ Since GDPR, honest reporting about user telemetry is required. The default track
 * COOKIE_SESS (sess,sid): The session id. Each time you visit/use the site its approximately broken into session ids.
 * COOKIE_JWT (jwt): The encrypted token of your user. This may optionally include your user id (uid) if logged in.
 
-
+### Pruning Records
+* Run  ```./tracker --prune config.json``` to run privacy pruning.
```

Comment:
Please clarify: This is something we will run manually every once in a while to trigger the pruning or is it the new regular startup command for the tracker processes inside the dockers?

Reply:
This is a once in a while (crontab I'm envisioning)